packages/common/src/PriceIdentifierUtils.js

```diff
@@ -25,8 +25,21 @@ const getPrecisionForIdentifier = identifier => {
   return IDENTIFIER_NON_18_PRECISION[identifier] ? IDENTIFIER_NON_18_PRECISION[identifier] : 18;
 };
 
+// The optimistic oracle proposer should skip proposing prices for these identifiers, for expired EMP contracts,
+// because they map to self-referential pricefeeds pre-expiry, but have different price resolution ogic post-expiry.
+// For example, please see [UMIP47](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-47.md):
+// - "The type of price that the DVM will return is dependent on the timestamp the price request is made at. This
+//   timestamp is the expiry timestamp of the contract that is intended to use this price identifier, so the TWAP
+//   calculation is used pre-expiry and the closing index value of uSTONKS calculation is used at expiry.""
+const OPTIMISTIC_ORACLE_IGNORE_POST_EXPIRY = [
```

Comment:
@smb2796 can you confirm these are the identifiers we want to blacklist?

Reply:
Yep these are the only identifiers needed for now. They are used in [uSTONKS](https://etherscan.io/address/0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C) contract and the [uGAS-JUN21](https://etherscan.io/address/0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43) contract.